src/model/entity/level_test.go

```diff
@@ -0,0 +1,64 @@
+package entity
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestLevel_GenerateRoomsOnLevel(t *testing.T) {
+	tests := []struct {
+		name          string
+		mapWidth      int
+		mapHeight     int
+		want          bool
```

Comment:
Что это значит? Я предлагаю такое название:

```suggestion
		wantErr          bool
```